src/proto/h1/role.rs

```diff
@@ -359,16 +384,51 @@ impl Http1Transaction for Server {
         let init_cap = 30 + msg.head.headers.len() * AVERAGE_HEADER_SIZE;
         dst.reserve(init_cap);
         if msg.head.version == Version::HTTP_11 && msg.head.subject == StatusCode::OK {
+            #[cfg(feature = "traceable")]
+            span.as_ref().unwrap().record("http.flavor", &1.1);
```

Comment:
This will be replaced by a proc-macro-function, such as `record_field!("http.flavor", &1.1)`